members/SWAU22467_226588.yaml

```diff
@@ -0,0 +1,72 @@
+bioguide: SWAU22467_226588
+contact_form:
+  driver: phantom
+  steps:
+    - visit: "https://app.leg.wa.gov/pbc/memberEmail/2/0"
+    - fill_in:
```

Comment:
maybe start it off with a `find`? looks like the page is fully server rendered so probably not strictly necessary, but couldn't hurt

Reply:
added it